toolbox/load_aware.py

```diff
@@ -58,9 +58,9 @@ def undeploy_trimaran(self):
 
     @AnsibleRole("load_aware_scale_test")
     @AnsibleMappedParams
-    def scale_test(self, distribution="poisson", duration=60.0, instances=10, namespace="load-aware"):
+    def scale_test(self, distribution="poisson", duration=60.0, instances=10, namespace="load-aware", scheduler="default"):
```

Comment:
you'll have to add a short description for all the parameters in the `Args` below